# HG changeset patch
# User John D <john.donoghue@ieee.org>
# Date 1501185333 14400
#      Thu Jul 27 15:55:33 2017 -0400
# Node ID 9d67bd90681b21b4b85269b97a37e676483bf0d4
# Parent  2fdc65c031b594c6aa2681c7e3351c4d91f3c314
cross tools: define OCTAVE_UNUSED if not defined

* src/mkoctfile.in.cc, src/octave-config.in.cc
  add #define OCTAVE_UNUSED if not already defined

diff --git a/src/mkoctfile.in.cc b/src/mkoctfile.in.cc
--- a/src/mkoctfile.in.cc
+++ b/src/mkoctfile.in.cc
@@ -35,16 +35,20 @@ along with Octave; see the file COPYING.
 #include <sstream>
 #include <vector>
 #include <cstdlib>
 
 #if defined (CROSS)
 #  include <sys/types.h>
 #  include <sys/wait.h>
 #  include <unistd.h>
+
+#  ifndef OCTAVE_UNUSED
+#    define OCTAVE_UNUSED
+#  endif
 #else
 #  include "unistd-wrappers.h"
 #  include "wait-wrappers.h"
 #endif
 
 static std::map<std::string, std::string> vars;
 
 #if ! defined (OCTAVE_VERSION)
diff --git a/src/octave-config.in.cc b/src/octave-config.in.cc
--- a/src/octave-config.in.cc
+++ b/src/octave-config.in.cc
@@ -34,16 +34,20 @@ along with Octave; see the file COPYING.
 #if ! defined (OCTAVE_PREFIX)
 #  define OCTAVE_PREFIX %OCTAVE_PREFIX%
 #endif
 
 #if ! defined (OCTAVE_EXEC_PREFIX)
 #  define OCTAVE_EXEC_PREFIX %OCTAVE_EXEC_PREFIX%
 #endif
 
+#ifndef OCTAVE_UNUSED
+#  define OCTAVE_UNUSED
+#endif
+
 #include "shared-fcns.h"
 
 static std::map<std::string,std::string> vars;
 
 static std::string usage_msg = "usage: octave-config [options]";
 
 static std::string help_msg =
 "\n"
